Gemfile

```diff
@@ -17,6 +17,7 @@ gem "minima", "~> 2.5"
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
   gem "jekyll-postcss"
+  gem "jekyll-purgecss"
```

Comment:
Instead of asking JS to do the purge, we'll get jekyll to handle it after we've built the site 👍